lib/cloudflare/custom_hostname/ssl_attribute/settings.rb

```diff
@@ -2,75 +2,75 @@
 
 module Cloudflare
 
-  class CustomHostname < Representation
```

Comment:
This file only has spaces -> tabs changes

Reply:
Thanks for being consistent.